Makefile

```diff
@@ -7,10 +7,15 @@ SHELL := /usr/bin/env bash
 TARGETOS ?= $(shell go env GOOS)
 TARGETARCH ?= $(shell go env GOARCH)
 PROJECT_NAME ?= llm-d-inference-scheduler
+SIDECAR_IMAGE_NAME ?= llm-d-routing-sidecar
```

Comment:
nit: ditto for naming conventions of variables in the Makfile (`EPP_` and `SIDECAR_`...)